api/src/main/resources/assets/nolijium/lang/zh_cn.json

```diff
@@ -83,13 +106,15 @@
     "nolijium.options.disable_tutorial_toasts.tooltip": "禁用教程提示。",
     "nolijium.particles": "粒子效果",
     "nolijium.options.disable_particles.name": "隐藏粒子",
-    "nolijium.options.disable_particles.tooltip": "防止所有粒子渲染。",
+    "nolijium.options.disable_particles.tooltip": "阻止所有粒子渲染。",
     "nolijium.particle_id": "允许 %s 粒子。",
     "nolijium.chroma": "彩色模式",
     "nolijium.options.chroma_speed.name": "色彩变化速度",
     "nolijium.options.chroma_speed.tooltip": "较高的值循环速度较快，较低的值循环速度较慢。",
     "nolijium.options.enable_chroma_block_outlines.name": "启用彩色方块轮廓",
     "nolijium.options.enable_chroma_block_outlines.tooltip": "启用方块轮廓上的彩色效果。",
+    "nolijium.options.chroma_block_shape_overlay.name": "启用彩色方块形状覆盖",
+    "nolijium.options.chroma_block_shape_overlay.tooltip": "如果设置，将在方块表面绘制彩色覆盖，使用指定值作为透明度。\n在启用§oFabulous!§r渲染时可能会在某些方块上出现异常。",
```

Comment:
This is copied from the vanilla quality level translation. Does the vanilla `zh_cn` translation not include this?

Reply:
Oh! It was an oversight on my part.